src/ui/render_config.rs

```diff
@@ -127,6 +127,12 @@ pub struct RenderConfig {
     /// a separator from the prefix.
     pub option: StyleSheet,
 
+    /// Style sheet for the option that is currently selected.
```

Comment:
Whether we go with the `Option<StyleSheet>` or just `StyleSheet`, I think it'll be important to detail how this config behaves with the `option` one.

Reply:
Done in https://github.com/mikaelmello/inquire/pull/88/commits/c701464b25e398d3b748c15e84ba5e1dc9b8ddbd